### PR TITLE
Feature/EPMUII-123

### PR DESCRIPTION
**Pull request by Dmitry-White**

### CODE REVIEW

File: app/libs/commonCrudOperations.js [hunk 1]
```diff
@@ -40,6 +40,19 @@ export const getById = (Model, findBy) => {
   };
 };
 
+export const removeById = (Model, findBy) => {
+  return (req, res, next) => {
+    const select = findBy
```
Comment (SergeyKovalchuk) [inline app/libs/commonCrudOperations.js]: The same as for https://github.com/FrontSpot-Community/code-verify/pull/53

**Review by SergeyKovalchuk — CHANGES_REQUESTED** [review]
Please fix this right after merge of https://github.com/FrontSpot-Community/code-verify/pull/53

File: app/routers/tournament.router.js [hunk 2]
```diff
@@ -9,6 +11,7 @@ const router = express.Router();
 router.get('/tournament', getAll);
 router.get('/tournament/:id', getById);
 router.post('/tournament', add);
+router.post('/tournament/:id', remove);
```
Comment (VladCherniavsky) [inline app/routers/tournament.router.js]: Please use `delete` instead of `post`

**Comment by SergeyKovalchuk** [comment]
Too many commits let's squash

**Comment by egor-miasnikov** [comment]
Also could you please use rebase against merge?

**Pull request merged**